docs/scos/dev/tutorials-and-howtos/introduction-tutorials/tutorial-sending-an-email.md

```diff
@@ -126,231 +126,159 @@ Create the `MailType` plugin for this example. For more information about creati
+                    ->setIsHtml(false),
+            )
+            ->addRecipient(
+                (new MailRecipientTransfer())
+                    ->setEmail($customerTransfer->getEmail())
+                    ->setName(sprintf('%s %s', $customerTransfer->getFirstName(), $customerTransfer->getLastName())),
+            );
     }
 }
 ```
-
 </details>
 
-The `Mail` module’s default `MailBuilder` is already pre-defined to build the `MailTransfer`. `MailBuilder` internally adds a new `MailRecipientTransfer` with the passed information, email, and name.
-
 ## 3. Registering a plugin
 
 When the plugin is created, it must be registered in `MailDependencyProvider`:
 
 ```php
 <?php
-        ...
-        $container->extend(self::MAIL_TYPE_COLLECTION, function (MailTypeCollectionAddInterface $mailCollection) {
-            ....
-            $mailCollection->add(new CustomCustomerRegistrationMailTypePlugin());
-
-            return $mailCollection
-        });
-        ...
- ?>
-```
-
-## 4. Mail translations
-
-The default `MailBuilder` also has access to the glossary with the `setSubject()` method. This is used for translations as follows:
-
-```php
-<?php
-namespace Pyz\Zed\Customer\Communication\Plugin\Mail;
+namespace Pyz\Zed\Mail;
 
-use Spryker\Zed\Kernel\Communication\AbstractPlugin;
-use Spryker\Zed\Mail\Business\Model\Mail\Builder\MailBuilderInterface;
-use Spryker\Zed\Mail\Dependency\Plugin\MailTypeInterface;
+use Pyz\Zed\Customer\Communication\Plugin\Mail\CustomCustomerRegistrationMailTypeBuilderPlugin;
+use Spryker\Zed\Mail\MailDependencyProvider as SprykerMailDependencyProvider;
 
-class CustomerCustomerRegistrationMailTypePlugin extends AbstractPlugin implements MailTypePluginInterface
+class MailDependencyProvider extends SprykerMailDependencyProvider
 {
-    ...
-    protected function setSubject(MailBuilderInterface $mailBuilder)
+    protected function getMailTypeBuilderPlugins(): array
     {
-        $mailBuilder->setSubject('mail.customer.registration.subject');
+        return [
+            new CustomCustomerRegistrationMailTypeBuilderPlugin(),
+        ];
     }
-    ...
 }
 ```
 
-A string is used as a key of the translation. The `MailBuilder` internally does the translation through `GlossaryFacade`:
-
-```php
-<?php
-namespace Spryker\Zed\Mail\Business\Model\Mail\Builder;
+## 4. Mail translations
 
-    ...
+The `MailTypeBuilderPlugin` also has access to the glossary with the `setSubject()` method.
 
-    protected function setSubject($subject, array $data = [])
-    {
-        $subject = $this->translate($subject, $data);
+A string is used as a key of the translation. The default mail provider internally does the translation through `GlossaryFacade`.
 
-        $this->getMailTransfer()->setSubject($subject);
+You can also translate with the parameters setting up the placeholder to be replaced.  For the `mail.order.shipped.subject` key, you have `Your order {orderReference} is on its way as translation`.
```

Comment:
Glossary translation might have parameters.
E.g. if we have the following glossary keys in the text for the email subject `Your order {orderReference} is on its way as translation`, whenever the `{orderReference}` parameter is provided, it will be replaced by the order reference.